lib/reversetunnel/agent.go

```diff
@@ -585,6 +588,19 @@ func (a *agent) handleChannels() error {
 				a.handleDiscovery(ch, req)
 				a.wg.Done()
 			}()
+		case nch := <-a.reconnectC:
+			if nch == nil {
+				continue
+			}
+
+			a.log.Debugf("Received reconnect from proxy.")
```

Comment:
```suggestion
			a.log.Debugf("Received reconnect advisory message from proxy.")
```
or something along those lines?